### PR TITLE
doc: Tune/Lightning notebook examples update title and add reference to AIR example

### PR DESCRIPTION
**Pull request by jimthompson5802**
## Why are these changes needed?

There are two example notebooks that show using Ray Tune with Torch Lightning.  One is based on AIR api set and the other uses "vanilla" api.  

The titles for both notebooks are the same.  This can cause confusion.  

This PR changes the title of the "vanilla" version to make it more distinguishable from the other notebook.  It also puts a link from the "vanilla" version to the AIR version, similar to the link from the AIR version to the "vanilla" version.

## Related issue number

N/A

## Checks

- [ ] I've signed off every commit(by using the -s flag, i.e., `git commit -s`) in this PR.
- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
    - [ ] I've added any new APIs to the API Reference. For example, if I added a 
           method in Tune, I've added it in `doc/source/tune/api/` under the 
           corresponding `.rst` file.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

**Review by krfricke — CHANGES_REQUESTED** [review]
Thanks for the contribution - we are currently doing some refactorings of the documentation, and one of them includes to Ray AIR (we are de-emphasizing the umbrella term in favor of the specific libraries).

We are also deprecating the LightningTrainer in favor of the regular PyTorch trainer. It would be great if you could reference to that trainer, and add a sentence explaining when people would want to use it (i.e., if they also require distributed training within each trial).

File: doc/source/tune/examples/tune-vanilla-pytorch-lightning.ipynb [hunk 1]
```diff
@@ -28,6 +28,11 @@
     "use it plug and play for your existing models, assuming their parameters are configurable!\n",
     "\n",
     ":::{note}\n",
+    "If you want to use AIR {class}`LightningTrainer <ray.train.lightning.LightningTrainer>`, please refer to this document: {ref}`Using Pytorch Lightning with Tune <tune-pytorch-lightning-ref>`.\n",
```
Comment (woshiyyya) [inline doc/source/tune/examples/tune-vanilla-pytorch-lightning.ipynb:31]:
```suggestion
    "If you want to use Ray Train's {class}`TorchTrainer <ray.train.torch.TorchTrainer>`, please refer to this user guide: {ref}`Getting Started with PyTorch Lightning <train-pytorch-lightning>`.\n",
```

**Review by woshiyyya — CHANGES_REQUESTED** [review]

**Review by woshiyyya — APPROVED** [review]
LGTM. Let's wait for CI to pass.

**Comment by jimthompson5802** [comment]
@woshiyyya @krfricke   Not sure but I may have made a mistake in trying address the DCO check.  I followed the instructions provided.  It appears I may have accidentally signed more than just my commit.

I'm thinking it might make sense to just close this PR and either start again or since you guys are refactoring the notebooks and the changes I proposed  are simple documentation one you can incorporate in your update.

Anyway let me know what I should do.

**Comment by woshiyyya** [comment]
Hi @jimthompson5802 , can you make another PR with the change only? 

(To pass the DCO check, you can use `git commit -m "XXX" -s`)

**Pull request closed without merging**